content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -33,9 +33,9 @@ This page describes how to use Admission Webhooks and Initializers.
 
 Admission webhooks are HTTP callbacks that receive admission requests and do
 something with them. You can define two types of admission webhooks,
-[validating admission Webhook](/docs/admin/admission-controllers/#validatingadmissionwebhook-alpha-in-18-beta-in-19)
+[validating admission Webhook](/docs/admin/admission-controllers/#validatingadmissionwebhook-alpha-in-1-8-beta-in-1-9)
 and
-[mutating admission webhook](/docs/admin/admission-controllers/#mutatingadmissionwebhook-beta-in-19).
+[mutating admission webhook](/docs/admin/admission-controllers/#mutatingadmissionwebhook-beta-in-1-9).
```

Comment:
Update to new link location as well - /docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook-beta-in-1-9